### ISSUE TITLE
'getter' functions

### ISSUE THREAD

**Issue by ciknowles** [cocopon/tweakpane#61]
Hi,

Sorry for yet another comment. This is an ease of use thing/enhancement suggestion.

When setting up my menu items I can specify properties such as min/max step etc like this

tweakpane.addInput(menu, 'value', x: {min:0, max: 100, step:0.1});

I don't think I can specify javascript getter functions

tweakpane.addInput(menu, 'value', x: {min:mymingetterfn, max: mymaxgetterfn, step:mystepfn});

This means I have to set up a javascript object for the menu. When changes occur in my interface copy these into the menu javascript object and call tweakpane.refresh. If functions were permitted, I could just call tweakpane.refresh?

Taking this further, I assume that 'value' could also be replaced optionally with a function? So the interface would look like:

tweakpane.addInput(contextobject, 'value'/fn, x: {min:value/fn, max: value/fn, step:value/fn});

Chris

**Comment by cocopon** [comment]
It is by design. I think we don't have to change value range at runtime in most cases. If you want to change them, dispose and re-create it with new config.

**Comment by ciknowles** [comment]
Hi, 

Ok, I understand the config issues with the View. 

For the value - it can be specified using a standard get/set javascript method. The get function can be crafted to pull the value from whatever you are controlling. This is pretty neat.

I've also found that the set function will be called when the value is changed (I've only tested a simple input though). This is an alternative to binding to the 'on' event.

Chris
